From 096b4606b1cebfc443d867981a86a2c73bf1ac4b Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 10 Nov 2018 19:04:01 +0100
Subject: [PATCH 1/1] configs: disable Unicode capitalization on VF610

For colibri_vf_defconfig u-boot-nodtb.bin exceeds file size limit.

Disable CONFIG_EFI_UNICODE_CAPITALIZATION for vf610-colibri_vf61 to save a
few KiB.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
Cf. https://travis-ci.org/xypron2/u-boot/jobs/453298845
---
 configs/colibri_vf_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/colibri_vf_defconfig b/configs/colibri_vf_defconfig
index 5854910f7ba..38f450c8eda 100644
--- a/configs/colibri_vf_defconfig
+++ b/configs/colibri_vf_defconfig
@@ -72,3 +72,4 @@ CONFIG_VIDEO_FSL_DCU_FB=y
 CONFIG_VIDEO=y
 CONFIG_SYS_CONSOLE_FG_COL=0x00
 CONFIG_OF_LIBFDT_OVERLAY=y
+# CONFIG_EFI_UNICODE_CAPITALIZATION is not set
-- 
2.19.1

